README.md

```diff
@@ -27,6 +27,8 @@ To build Spark and its example programs, run:
 (You do not need to do this if you downloaded a pre-built package.)
 More detailed documentation is available from the project site, at
 ["Building Spark"](http://spark.apache.org/docs/latest/building-spark.html).
+For developing Spark using an IDE, see [Eclipse](https://cwiki.apache.org/confluence/display/SPARK/Useful+Developer+Tools#UsefulDeveloperTools-Eclipse)
```

Comment:
I as a (lazy) developer would prefer links to 'my' IDE. What I prefer even more would be the instructions directly in README.md (or in a separate DEVELOPING.md file). This would make it much easier to keep them updated as you don't need yet another account for the wiki (at least the instructions for Eclipse refer to outdated versions).
What do you think?